src/devtools/client/webconsole/reducers/messages.js

```diff
@@ -132,7 +130,9 @@ function addMessage(newMessage, state, filtersState) {
     maybeSortVisibleMessages(state);
   } else if (DEFAULT_FILTERS.includes(cause)) {
     state.filteredMessagesCount.global++;
-    state.filteredMessagesCount[cause]++;
+  }
+  if (addedMessage.level) {
```

Comment:
This changes the `filteredMessagesCount` so that we now always count the # of messages that match this filter as opposed to only counting the truly filtered items.